### ISSUE TITLE
Printing problem in Firefox

### ISSUE THREAD

**Issue by hjshust** [i-rinat/freshplayerplugin#338]
Hi,

I have a question as below when I use this plug-in on my own Aarch64 machine. I appreciate it if it gets printing...

1. Environments
CPU: Aarch64
OS:  ubuntu 14.10
Flash: Chromebook 32bits PPAPI Flash
Brower: FireFox 32bits, version43

2: issue
With this fresh pepper plug-in, the Chrome PPAPI flash works well in Firefox, it plays videos or forms/lists well,however, it prints nothing if we try to click the right button of mouse, even if it does NOT show any printing menu or pop any setting windows. Any idea about it?

Thank you!

**Comment by i-rinat** [comment]
Printing is not implemented at the moment. There are just stub functions that do nothing.

**Comment by hjshust** [comment]
Thanks for the prompt reply... So do we have any plan for the printing implementation, when or in which version? Is there any open souce code that we can refer to at the moment to implement it, how complicated it is we can estimate?

Sorry for so many questions...

Best Regards,

**Comment by i-rinat** [comment]
> have any plan for the printing implementation

Yes, I do have plans to implement printing support as well. But there is no strict schedule, so I can't say when. Lately, I just bump version after a number of months, so can't say which version will printing be implemented in.

> Is there any open souce code that we can refer to at the moment to implement it, how complicated it is we can estimate?

It depends. I haven't started to look at it yet. But obvious pointers are Firefox and Chromium source code.
